core/templates/domain/skill/skill-update.service.spec.ts

```diff
@@ -355,107 +441,139 @@ describe('Skill update service', function() {
         content_id: 'worked_example_e_3'
       }
     };
-    SkillUpdateService.addWorkedExample(
-      skill, workedExampleObjectFactory.createFromBackendDict(newExample));
-    expect(UndoRedoService.getCommittableChangeList()).toEqual([{
+
+    skillUpdateService.addWorkedExample(
+      skill,
+      workedExampleObjectFactory.createFromBackendDict(newExample)
+    );
+    const workedExamplesObject = {
       cmd: 'update_skill_contents_property',
       property_name: 'worked_examples',
-      old_value: [example1, example2],
+      old_value: skillContentsDict.worked_examples,
       new_value: [example1, example2, newExample]
-    }]);
+    };
+
+    expect(undoRedoService.getCommittableChangeList()).toEqual(
+      // This throws No overload matches this call.
```

Comment:
In general, it is better to fix a problem than ignore it. I think we should fix it by passing the right type, and assigning all the properties.

Reply:
Can you try doing this everywhere in this file? Let me know if you need my help at any point

Reply:
It only throws this error for workedExamples. I need your help